src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+
+    This service account will be referred to as the _folder-management service account_ in the remainder of this page.
+
+### Grant the `FOLDER_ADMIN` role to the service account
+
+Folders inherit roles granted at the organization scope, and descendent folders inherit roles granted higher in the hierarchy.
+
+During Limited Access, it is not possible to use the {{ site.data.products.db }} Console to grant the Folder Admin or Folder Mover roles.
+Instead, use the CockroachDB {{ site.data.products.db }} API and the provisioning service account you created in [Gather required information](#gather-required-information) to grant the folder-management service account the `FOLDER_ADMIN` role at the level of the organization.
+
+1. To grant the role at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<folder-management-service-account-id>/ORGANIZATION/<organization_id>/FOLDER_ADMIN' \
+           --header 'Authorization: Bearer <provisioning_service_account_api-key>'
+    ~~~
+
+    Replace:
+    - `<folder-management-service-account-id>`: the ID of the user or service account being granted the role.
+    - `<organization_id>`:your organization's ID.
+    - `<provisioning_service_account_api-key>`: The API key of a service account with the Org Admininstrator role.
+
+    After your desired folder structure is in place, you could revoke the role from the organization and grant it on individual folders instead. A recommended security practice is to limit the users or service accounts with `FOLDER_ADMIN` at the level of the organization. When you [create a folder](#create-a-folder-and-grant-access-to-it), you can grant the `FOLDER_ADMIN` role at the level of the folder.
+
+1. To revoke the role at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request DELETE \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<user-id>/ORGANIZATION/<organization_id>/FOLDER_ADMIN' \
+           --header 'Authorization: <api-key>'
+    ~~~
+
+    Replace:
+    - `<user-id>`: the ID of the user or service account from whom to revoke the role.
+    - `<organization_id>`:your organization's ID.
+    - `<api-key>`: The API key of a service account with the Org Admininstrator role.
+
+1. Optionally, you can now delete the provisioning service account. In the remainder of this page, replace `<api-key>` with the API key for the folder-management service account.
+
+## Create a folder and grant access to it
+
+Your service account must have the following roles on the organization, the folder, or by inheritance:
+
+- [Folder Admin](authorization.html#folder-admin)
+
+1. To create a folder at the organization level, set its `parent_id` field to `root`:
```

Comment:
nit: also note that they can leave the `parent_id` field blank

Reply:
Done